src/main/java/at/hannibal2/skyhanni/features/misc/CollectionTracker.kt

```diff
@@ -193,10 +226,22 @@ object CollectionTracker {
 
     @HandleEvent
     fun onTick() {
+        if (isDungeonBoss) return
         compareInventory()
         updateGain()
     }
 
+    @HandleEvent
+    fun onDungeonComplete(event: DungeonCompleteEvent) {
+        if (isDungeonBoss) {
+            val foundAmount = ProfileStorageData.profileSpecific?.dungeons?.bosses[dungeonFloor]
```

Comment:
Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type MutableMap<DungeonFloor, Int>?

Reply:
Type mismatch: inferred type is DungeonFloor? but DungeonFloor was expected